compiler/wasm/src/compile.rs

```diff
@@ -164,10 +164,9 @@ pub fn compile_program(
     console_error_panic_hook::set_once();
     let (crate_id, mut context) = prepare_context(entry_point, dependency_graph, file_source_map)?;
 
-    let compile_options = CompileOptions {
-        expression_width: ExpressionWidth::Bounded { width: 4 },
-        ..CompileOptions::default()
-    };
+    let expression_width = ExpressionWidth::Bounded { width: 4 };
```

Comment:
Can you open an issue for us to make this customisable? Not a blocker for this PR as expression width was already fixed.

Reply:
https://github.com/noir-lang/noir/issues/5509